aries-site/src/themes/aries.js

```diff
@@ -6,10 +6,42 @@ export const aries = deepMerge(hpe, {
   // To be stripped out once theme changes are made in grommet-theme-hpe
   // keeping file for use as playground for future theme adjusments that need
   // to be quickly tested
-  menu: {
-    group: {
-      separator: {
-        color: 'border-weak',
+  feedback: {
+    body: {
+      gap: 'medium',
+    },
+    button: {
+      a11yTitle: `You are in a layer containing a form to submit feedback.
+        To close this layer, press Enter.`,
+    },
+    container: {
+      pad: 'medium',
+    },
+    header: {
+      size: 'small',
+      level: 2,
+      margin: {
+        vertical: 'none',
+      },
+      container: {
+        align: 'start',
+        direction: 'row',
+        justify: 'between',
+      },
+    },
+    footer: {
+      pad: {
+        vertical: 'medium',
+      },
+      direction: 'row',
+      justify: 'end',
+      gap: 'medium',
+      success: {
```

Comment:
wondering if we pull `success` out of the `footer` object so it's just:

```
feedback: {
   success: {
   // any text props
   }
}
```
then we can eliminate the `text` object.